.github/workflows/stackhpc-all-in-one.yml

```diff
@@ -335,6 +335,26 @@ jobs:
         env:
           KAYOBE_AUTOMATION_SSH_PRIVATE_KEY: ${{ steps.ssh_key.outputs.ssh_key }}
 
+      - name: Change RabbitMQ queues from transient types to durable types (quorum and streams)
+        run: |
+          sed -i -e '$aom_enable_queue_manager: True' \
+                 -e '$aom_enable_rabbitmq_quorum_queues: True' \
+                 -e '$aom_enable_rabbitmq_transient_quorum_queue: True' \
+                 -e '$aom_enable_rabbitmq_stream_fanout: True' \
+                 etc/kayobe/environments/ci-aio/kolla/globals.yml
+        if: inputs.upgrade
+
+      - name: Migrate RabbitMQ queues
+        run: |
+          docker run -t --rm \
+            -v $(pwd):/stack/kayobe-automation-env/src/kayobe-config \
+            -e KAYOBE_ENVIRONMENT -e KAYOBE_VAULT_PASSWORD -e KAYOBE_AUTOMATION_SSH_PRIVATE_KEY \
+            ${{ steps.kayobe_image.outputs.kayobe_image }} \
+            /stack/kayobe-automation-env/src/kayobe-config/.automation/pipeline/script-run.sh tools/rabbitmq-queue-migration.sh
+        env:
+          KAYOBE_AUTOMATION_SSH_PRIVATE_KEY: ${{ steps.ssh_key.outputs.ssh_key }}
+        if: inputs.upgrade
```

Comment:
We should probably wrap these up into an [upgrade-prerequisites.sh](https://github.com/stackhpc/stackhpc-kayobe-config/blob/stackhpc/2023.1/tools/upgrade-prerequisites.sh) script like we did for antelope, so it's easier to extend and can be used in Multinode automation

Reply:
Done